packages-exp/auth-types-exp/index.d.ts

```diff
@@ -446,7 +446,7 @@ export interface User extends UserInfo {
  */
 export interface UserCredential {
```

Comment:
woof. big api change, no?

Reply:
No this was actually in the plan all along; we had been following the old way instead of the new one. Go double check the doc :stuck_out_tongue:

Reply:
oh that is excellent